guide/case-studies/cloud-backup.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: guide
+title: Daily spending
+nav_order: 1
+parent: Case studies
+permalink: /guide/case-studies/cloud-backup/
+main_classes: -no-top-padding
+image: /assets/images/guide/getting-started/getting-started-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+-->
+
+## Cash account / Daily spending
+
+Imagine a product which tries to solve the problem of quickly and easily sending smaller amounts of money to friends and family, or for small purchases. Ease and speed of use will be important as usage is likely to be on mobile devices and on the go. Users are not expected to be well versed in bitcoin technology or advanced private key management, which makes it reasonable to worry more about self-inflicted loss than from theft.
+
+A single-key scheme with [automatic cloud backup]({{ '/guide/private-key-management/single-user-schemes/#automatic-cloud-backup' | relative_url }}) might be the best choice for the majority of users in this case. For more advanced users you could offer the option to opt-out of automatic cloud backup and instead use a recovery phrase.
+
+{% include prototype.html
+   link = "https://www.figma.com/proto/SRWlaxbDulsacpPQn2TTri/Case-study-prototypes?node-id=1%3A3&viewport=1357%2C576%2C1&scaling=scale-down"
+   image = "/assets/images/guide/case-studies/case-cash.png"
```

Comment:
Might be for another PR and I think it's been mentioned before, but should we start using SVG's instead so we can simplify and use one image now?

Reply:
Good call. We should do a review of this across the whole guide before we go live with V1.